objects/omapcamera/Create_0.gml

```diff
@@ -0,0 +1,10 @@
+camWidth = 160;
+camHeight = 120;
+
+follow = oTestPlayer;
```

Comment:
Use descriptive variable names, outside of this context "follow" could mean dozens of things.